packages/mermaid/src/defaultConfig.ts

```diff
@@ -50,6 +50,7 @@ const config: Partial<MermaidConfig> = {
     ...defaultConfigJson.gantt,
     tickInterval: undefined,
     useWidth: undefined, // can probably be removed since `configKeys` already includes this
+    weekday: 'sunday', // the sane default is a monday, but it's set to sunday for legacy reasons
```

Comment:
Please update the [`config.schema.yaml`](https://github.com/mermaid-js/mermaid/blob/666df2218ba9f7e543def057c8821492ca71f45d/packages/mermaid/src/schemas/config.schema.yaml) instead of this.